code/datums/storage/storage.dm

```diff
@@ -355,37 +355,37 @@ GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 
 	if(total_weight > max_total_storage)
 		if(messages && user && !silent_for_user)
-			to_chat(user, span_warning("\The [to_insert] can't fit into \the [resolve_parent]! Make some space!"))
+			user.balloon_alert(user, "no room!")
 		return FALSE
 
 	if(length(can_hold))
 		if(!is_type_in_typecache(to_insert, can_hold))
 			if(messages && user)
-				to_chat(user, span_warning("\The [resolve_parent] cannot hold \the [to_insert]!"))
+				user.balloon_alert(user, "can't hold!")
 			return FALSE
 
 	if(is_type_in_typecache(to_insert, cant_hold) || HAS_TRAIT(to_insert, TRAIT_NO_STORAGE_INSERT) || (can_hold_trait && !HAS_TRAIT(to_insert, can_hold_trait)))
 		if(messages && user)
-			to_chat(user, span_warning("\The [resolve_parent] cannot hold \the [to_insert]!"))
+			user.balloon_alert(user, "can't hold!")
 		return FALSE
 
 	if(HAS_TRAIT(to_insert, TRAIT_NODROP))
 		if(messages)
-			to_chat(user, span_warning("\The [to_insert] is stuck on your hand!"))
+			user.balloon_alert(user, "stuck on your hand!")
 		return FALSE
 
 	var/datum/storage/biggerfish = resolve_parent.loc.atom_storage // this is valid if the container our resolve_parent is being held in is a storage item
 
 	if(biggerfish && biggerfish.max_specific_storage < max_specific_storage)
 		if(messages && user)
-			to_chat(user, span_warning("[to_insert] can't fit in [resolve_parent] while [resolve_parent.loc] is in the way!"))
+			user.balloon_alert(user, "[resolve_parent.loc] is in the way!")
```

Comment:
```suggestion
			user.balloon_alert(user, "[lowertext(resolve_parent.loc.name)] is in the way!")
```

Reply:
--> "The large satchel is in the way!" -> "large satchel is in the way!"

Reply:
Or more ideally, just `"something is in the way!"` if it's obvious what it'd be. We don't like to use names in balloon alerts because they're inconsistent and usually pretty long.

Reply:
personally I feel like having the name here is fine, it might get a little long sometimes but I feel like if I were to just do "something is in the way!" or "cant reach!" it might get too confusing. I know I would get confused anyway. If you disagree Ill change it though.

Reply:
The question is is it obvious what is in the way? We have these standards for a reason and individual usages really should not pick and choose